pkg/flight/wasi/malloc.go

```diff
@@ -2,9 +2,9 @@
 // It exports a "malloc" func to let hosts allocate memory within the wasm module.
 package wasi
 
-import "github.com/yokecd/yoke/internal/wasm"
+// import "github.com/yokecd/yoke/internal/wasm"
 
 //go:wasmexport malloc
-func malloc(size uint32) wasm.Buffer {
-	return wasm.FromSlice(make([]byte, size))
-}
+// func malloc(size uint32) wasm.Buffer {
```

Comment:
This cannot be commented out. What's going on here? 😄

Reply:
Oh I see! Although it is "unused", it has a `//go:wasmexport` directive, which exports this function so that it can be called by yoke's host function to allocate memory in the wasm module.

TLDR: although its not used directly, it is absolutely used just not in a way that the linter can detect!
See: https://github.com/yokecd/yoke/blob/34f0acaa59c537a05ae67aef154b81e669915957/internal/wasi/wasi.go#L216-L219